repositories.yaml

```diff
@@ -302,3 +302,12 @@ mozphab:
     - 'mozphab/metrics.yaml'
   ping_files:
     - 'mozphab/pings.yaml'
+firefox-desktop:
+  app_id: firefox-desktop
+  url: 'https://github.com/mozilla/gecko-dev'
+  notification_emails:
+    - chutten@mozilla.com
+  metrics_files:
+    - 'toolkit/components/glean/ipc/metrics.yaml'
```

Comment:
```suggestion
    - 'toolkit/components/glean/metrics.yaml'
```

Reply:
I have no idea why I thought these were in `ipc`!